content/en/docs/concepts/services-networking/topology-aware-hints.md

```diff
@@ -0,0 +1,154 @@
+---
+reviewers:
+- robscott
+title: Topology Aware Hints
+content_type: concept
+weight: 10
+---
+
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.21" state="alpha" >}}
+
+_Topology Aware Hints_ enable topology aware routing with topology hints
```

Comment:
This is self-referential. "X enables X".

Maybe "_Topology Aware Hints_ enable topology aware routing by including suggestions for how clients should consume endpoints" or something?

Reply:
Updated, thanks!